monitoring/cluster-monitoring/configuring-the-monitoring-stack.adoc

```diff
@@ -27,13 +27,19 @@ include::modules/monitoring-moving-monitoring-components-to-different-nodes.adoc
 
 Running cluster monitoring with persistent storage means that your metrics are stored to a Persistent Volume and can survive a pod being restarted or recreated. This is ideal if you require your metrics or alerting data to be guarded from data loss. For production environments, it is highly recommended to configure persistent storage.
 
+[IMPORTANT]
+====
+In {product-title} 4.1 deployed on bare metal, Prometheus and Alertmanager cannot have persistent storage and are ephemeral only. For use cases other than bare metal, you must use dynamic provisioning to use persistent storage.
```

Comment:
s/4.1/{product-version}

Is this just bare metal, or is it vSphere, too?

Why can't you have persistent storage for it?